### DISCUSSION TITLE
Auto-extract duration for audio/video files

### DISCUSSION THREAD

**Discussion by scouech** [directus/directus#4420] [Feature Requests]
1) The file duration (here for mp3 file) is null

2) After uploading an mp3 file, if I try to retrieve the file, the duration field is `null`

3) I'm on the latest version of directus (via digital ocean marketplace)

**Comment by scouech** [comment]
Issue was moved to discussion, so I wonder if anyone knows about this? Thanks!

**Reply by benhaynes** [reply]
I think because this is _technically_ considered a new feature. As of now, we only track the `duration` for video files (and embeds). I agree that it would be expected for this to also track the duration of audio files, so this will certainly be implemented! We're just trying to keep our issues (bugs) and discussions (feature requests) organized. 🙂

**Comment by sGerli** [comment]
This would be great, I'm currently migrating an app with podcast hosting features to directus-9, and I'm missing this feature for the podcast mp3s.

**Reply by sGerli** [reply]
PR: https://github.com/directus/directus/pull/6865

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
